### DISCUSSION TITLE
Does the author have any consideration for linux?

### DISCUSSION THREAD

**Discussion by mingtianshi** [winsiderss/systeminformer#1213] [Questions / Answers]

**Comment by iam-py-test** [comment]
Probably hard, but a good process management tool on Linux would be nice

**Comment by Biswa96** [comment]
There are plenty of them for GNU/Linux environment.

**Comment by iam-py-test** [comment]
> There are plenty of them for GNU/Linux environment.

Which ones?

**Comment by Biswa96** [comment]
> Which ones?

* Linux kernel itself - /dev, /proc, /sys etc.
* Command line tools - ps, sysstat, lsof etc.
* TUI tools - top, htop, btop, ctop, gtop etc.
* GUI tools - gnome-system-monitor, plasma-systemmonitor, deepin-system-monitor etc.

**Comment by DavidXanatos** [comment]
@Biswa96 please excuse my harsh words but all the Task Managers for Linux (included all the once you named here) Suxxxxx donkey balls, bit times, they have a terrible user experience, bordering in being unusable!
The GUI once you mentioned have even less features than windows Xp's built in task manager.

You can't browse stack trace, open files, sockets, etc... you can not do anything more advanced with the UI once then list processes and kill them.
And the console once, well they are a pain to use in comparison with what you can do when you have an actual GUI.

**Comment by mingtianshi** [comment]
so what you have tools,if in the ubuntu @DavidXanatos I need your recommendation

**Comment by DavidXanatos** [comment]
I use linux rearly enough that i can make do without managing tasks most of the time, when i was considdering moving away from windows, i was considdering adding linux support to task explorer but as i decided i'll stuck to dinwos and instead make an own window privacy tool i dropped that idea, but if there are linux users willing to put in som work the sources are implemented soch that adding linux support does not need any refactoring only fill in all the empty abstract for linux classes that were planed but never implemented.

**Comment by as-muncher** [comment]
I have to agree with @DavidXanatos in that the gnome-system monitor tool and the one that comes with KDE are both garbage.  They don't work right.  They don't show nearly as much information as System Informer.  I for one would really love to see something for linux, and that's why I'm here, looking at your github page, specifically for this issue.
